client/client-lib/src/mint/mod.rs

```diff
@@ -205,6 +196,15 @@ impl MintClient {
             }
         }
 
+        let mut change_outputs: Vec<(usize, NoteIssuanceRequests)> = vec![];
+        for amount in change {
+            let (issuances, nonces) = self.create_ecash(amount).await;
+            let out_idx = tx.outputs.len();
+            tx.outputs.push(Output::Mint(MintOutput(nonces)));
+            change_outputs.push((out_idx, issuances));
+        }
+        let txid = tx.tx_hash();
+
```

Comment:
Moving this code block is merely cosmetic, right? Or did I miss a functional change?

Reply:
Necessary because it changes the result of `coins()`, i.e. the ecash we lose through spending.